05-requests/lowdefy.yaml

```diff
@@ -0,0 +1,81 @@
+lowdefy: 4.0.0-alpha.37
+name: Lowdefy starter
+
+menus:
+  - id: default
+    links:
+      - id: welcome
+        type: MenuLink
+        properties:
+          icon: AiOutlineHome
+          title: Home
+        pageId: welcome
+      - id: new-ticket
+        type: MenuLink
+        properties:
+          icon: AiOutlineAlert
+          title: New ticket
+        pageId: new-ticket
+
+pages:
+  - _ref: new-ticket.yaml
+
+  - id: welcome
+    type: PageHeaderMenu
+    properties:
+      title: Welcome
+    areas:
+      content:
+        justify: center
+        blocks:
+          - id: content_card
+            type: Card
+            style:
+              maxWidth: 800
+            blocks:
+              - id: content
+                type: Result
+                properties:
+                  title: Hello!
+                  subTitle: We are excited to see what you are going to build
+                  icon:
+                    name: AiOutlineHeart
+                    color: '#f00'
+                areas:
+                  extra:
+                    blocks:
+                      - id: docs_button
+                        type: Button
+                        properties:
+                          size: large
+                          title: Let's build something
+                          color: '#1890ff'
+                        events:
+                          onClick:
+                            - id: link_to_docs
+                              type: Link
+                              params:
+                                url: https://docs.lowdefy.com
+                                newTab: true
+      footer:
+        blocks:
+          - id: footer
+            type: Paragraph
+            properties:
+              type: secondary
+              content: |
+                Made by a Lowdefy 🤖
+
+connections:
```

Comment:
We normally add connections above pages, maybe it is better to do that here?

Reply:
Moved it above menus, as done in the v3 tutorial. Change made in ae20a454666402b2b3f545a9f05ef7985c7f50e4